src/raft_server.cpp

```diff
@@ -80,6 +83,10 @@ int ReplicationState::start(const butil::EndPoint & peering_endpoint, const int
 
     // flag controls snapshot download size of each RPC
     braft::FLAGS_raft_max_byte_count_per_rpc = snapshot_max_byte_count_per_rpc;
+    braft::FLAGS_raft_max_threads_snapshot_copy = snapshot_max_threads_per_copy;
+
+    braft::FLAGS_raft_rpc_channel_connect_timeout_ms = 2000;
+    braft::FLAGS_raft_max_threads_snapshot_copy = snapshot_max_threads_per_copy;
```

Comment:
@krunal1313 `FLAGS_raft_max_threads_snapshot_copy` duplicates. 

Just compile everything once locally to ensure that the merge conflicts are handled without errors.